<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>A9_windowClass</class>
 <widget class="QMainWindow" name="A9_windowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>A9_window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="admin_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="user_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="csv_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CSV</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="html_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HTML</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>474</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="A9_window.qrc"/>
 </resources>
 <connections/>
</ui>
